core/templates/components/skill-selector/skill-selector.directive.html

```diff
@@ -1,21 +1,63 @@
-<div class="list-group">
-  <md-card layout="row" class="list-view-item protractor-test-skill-select-header">
-    Search By Skill
-    <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" ng-model="skillFilterText" autofocus>
-  </md-card>
-  <div style="max-height: 50vh; overflow-y: auto;">
-    <a class="list-group-item protractor-test-skills-list-item"
-       ng-repeat="skillSummary in skillSummariesInitial | filter:skillFilterText"
-       ng-click="selectSkill(skillSummary.id)"
-       ng-class="{'list-group-item-success': (skillSummary.id === selectedSkillId)}">
-      <label class="list-group-item-heading"><[skillSummary.description]></label>
-    </a>
-    <hr ng-if="skillSummariesInitial.length > 0">
-    <a class="list-group-item protractor-test-skills-list-item"
-       ng-repeat="skillSummary in skillSummariesFinal | filter:skillFilterText"
-       ng-click="selectSkill(skillSummary.id)"
-       ng-class="{'list-group-item-success': (skillSummary.id === selectedSkillId)}">
-      <label class="list-group-item-heading protractor-test-skills-list-item"><[skillSummary.description]></label>
-    </a>
+<div class="skill-container" style="display:flex;  flex-direction: row;">
+  <div class="list-group" style="width:70%">
+    <md-card layout="row" class="list-view-item protractor-test-skill-select-header">
+      Search By Skill
+      <input type="text" class="form-control protractor-test-skill-name-input" placeholder="Enter a skill name" ng-model="skillFilterText" autofocus>
+    </md-card>
+    <div style="max-height: 50vh; overflow-y: auto;">
+      <md-radio-group ng-model="selectedSkill" ng-change="setSelectedSkillId()">
+        <md-card ng-if="!hasUntriagedSkills(topicName)" ng-repeat="(topicName, subtopics) in categorizedSkills" class="list-view-item">
+          <md-card layout="row" class="list-view-item">
+            <h2><label style="padding-left:10px"><[topicName]></label></h2>
+          </md-card>
+          <div ng-if="!checkIfEmpty(skills)" ng-repeat="(subtopicName, skills) in subtopics" class="list-view-item ">
+            <md-card class="list-view-item">
+              <label style="padding-top:20px;padding-bottom:20px;padding-left:10px"><[subtopicName]></label>
+              <md-radio-button style="margin-left:10px" class="protractor-test-skills-list-item" ng-repeat="skill in skills | filter:skillFilterText" ng-value="skill.skill_id" aria-label="<[skill.skill_description]>">
+                <[skill.skill_description]>
+              </md-radio-button>
+            </md-card>
+          </div>
+        </md-card>
+        <md-card ng-if="hasUntriagedSkills(topicName)" ng-repeat="(topicName, skills) in categorizedSkills" class="list-view-item">
+          <md-card layout="row" class="list-view-item">
+            <h2><label style="padding-left:10px"><[topicName]></label></h2>
```

Comment:
Not sure if this is the line, but in the modal the untriaged skills header is 'untriaged_skills' which looks weird as a display name usually don't have underscores, can you display the title for that section as 'Untriaged Skills'?

Reply:
done